docs/securing_installation.md

```diff
@@ -0,0 +1,108 @@
+# Securing your Helm Installation
+
+Helm is a powerful and flexible package-management and operations tool for Kubernetes. Installing it using the default installation command -- `helm init` -- quickly and easily installs **Tiller**, the server-side component with which Helm corresponds. 
+
+This default installation applies **_no security configurations_**, however. It's completely appropriate to use this type of installation when you are working against a cluster with no or very few security concerns, such as local development with Minikube or with a cluster that is well-secured in a private network with no data-sharing or no other users or teams. If this is the case, then the default installation is fine, but remember: With great power comes great responsibility. Always use due diligence when deciding to use the default installation.
+
+## Who Needs Security Configurations?
+
+For the following types of clusters we strongly recommend that you apply the proper security configurations to Helm and Tiller to ensure the safety of the cluster, the data in it, and the network to which it is connected.
+
+- Clusters that are exposed to uncontrolled network environments
+- Clusters that are for many people to use -- _multitenant_ clusters -- as a shared environment
+- Clusters that have access to or use high-value data or networks of any type
+
+Often, environments like these are referred to as _production grade_ or _production quality_ because the damage done to any company by misuse of the cluster can be profound for either customers, the company itself, or both. Once the risk of damage becomes high enough, you need to ensure the integrity of your cluster no matter what the actual risk. 
+
+To configure your installation properly for your environment, you must:
+
+- Understand the security context of your cluster
+- Choose the Best Practices you should apply to your helm installation
+
+The following assumes you have a Kubernetes configuration file (a _kubeconfig_ file) or one was given to you to access a cluster. 
+
+## Understanding the Security Context of your Cluster
+
+`helm init` installs Tiller into the cluster in the `default` namespace and without any RBAC rules applied. This is appropriate for local development and other private scenarios because it enables you to be productive immediately. It also enables you to continue running Helm with existing Kubernetes clusters that do not have role-based access control (RBAC) support until you can move your workloads to a more recent Kubernetes version.
```

Comment:
`helm init` installs tiller in the **kube-system** namespace by default.

Reply:
ACK. good catch.